src/main/features/orders/views/confirmation.njk

```diff
@@ -0,0 +1,35 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import submitButton, errorSummary %}
+{% from "support.njk" import supportWidget %}
+{% from "internalLink.njk" import internalLink %}
+
+{% set headingVisible = false %}
+{% set heading = 'You’ve asked the court to review the order' %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+
+      <div class="govuk-box-highlight">
+        <h1 class="bold-large">{{ t('You’ve asked the court to review the order') }}</h1>
+        <p class="receipt-download-container">
+          {{ internalLink(t('Download request for review of order'), OrdersPaths.reviewOrderReceiver.evaluateUri({ externalId: claim.externalId})) }}(PDF)
```

Comment:
the (PDF) needs to be in a t so it should be {{ t('(PDF)') }}

Reply:
done